components/plurals/src/lib.rs

```diff
@@ -258,6 +258,18 @@ impl PluralCategory {
             _ => None,
         }
     }
+    /// Returns the PluralCategory coresponding to given TR35 string as bytes
+    pub fn from_tr35_bytes(category: &[u8]) -> Option<PluralCategory> {
```

Comment:
Suggestion: leverage the same code path by calling from_tr35_bytes from from_tr35_string

Reply:
oh oops, meant to do that and forgot